stock_inventory/models/stock_quant.py

```diff
@@ -9,6 +9,7 @@ class StockQuant(models.Model):
     def _apply_inventory(self):
         res = super()._apply_inventory()
         record_moves = self.env["stock.move.line"]
+        adjustment = False
```

Comment:
```suggestion
        adjustment = self.env["stock.inventory"].browse()
```

is semantically more correct

Reply:
Sure! Just did that. Thanks